stripe-payment-link-sms/assets/index.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Get started with your Twilio Functions!</title>
+    <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Roboto:300,300italic,700,700italic">
+    <link rel="stylesheet" href="https://unpkg.com/normalize.css/normalize.css">
+    <link rel="stylesheet" href="https://unpkg.com/milligram/dist/milligram.min.css">
+    <style>
+      body {
+        padding: 20px;
+        display: flex;
+        flex-direction: column;
+        min-height: 100vh;
+      }
+
+      div[role="main"] {
+        flex: 1;
+      }
+
+      footer {
+        text-align: center;
+      }
+
+      footer p {
+        margin-bottom: 0;
+      }
+
+      #twilio-logo {
+        width: 50px;
+        height: 50px;
+      }
+    </style>
+  </head>
+  <body>
+    <header>
+      <h1>Welcome to your new Twilio App</h1>
+    </header>
+    <div role="main">
+      <section>
+        <h3>Get started with your app</h3>
+        <p>This app shows you how to create an invoice in Stripe based on an SMS prompt (in this case `DONATE <AMOUNT>`). Once the invoice is finalized, it will reply with a payment link.</p>
+        <ol>
+          <li>Before you can test your app, you need to add a <a href="https://dashboard.stripe.com/webhooks"> Stripe Webhook</a> and set the URL to the URL of this page, but replace `/index.html` with `/send-invoice-sms`</li>
```

Comment:
`> Stripe Webhook</a>` has an extra space before `Stripe`

Reply:
```
`/index.html` with `/send-invoice-sms`
```
is markdown and not html

Reply:
There is also a short cut so you don't have to do this. You can instead write:

```suggestion
<li>Before you can test your app, you need to add a <a href="https://dashboard.stripe.com/webhooks"> Stripe Webhook</a> and set it to <code><span class="function-root"></span>/send-invoice-sms</code></li>
```

The span part will automatically be replaced with the right part of the URL